multi-model-endpoint/deploy-endpoint.yml

```diff
@@ -40,7 +40,7 @@ Resources:
         Ref: Stage
       Name: !Sub ${ModelName}-api-${Stage}
       Domain: 
-        DomainName: api.dev.detektivkollektiv.org
+        DomainName: !Sub api.${Stage}.detektivkollektiv.org
```

Comment:
@ML-Dirk This is going to cause issues in the prod stage where the endpoint should be `api.detektivkollektiv.org` and not `api.prod.detektivkollektiv.org`.